### DISCUSSION TITLE
Filecoin staking?

### DISCUSSION THREAD

**Discussion by Jackyap84** [filecoin-project/community#380] [General] [answered]
Dear Filecoin community, i came across the FB ads which offer great staking APY, just wanna know if its legit or not, below is the link

https://filbc.io/?ref=17941212&fbclid=IwAR0uR7lYGJkzUWkvY8tWRcIX9N9qpk_ll9lD_8RvjpQ0EMH5a2_YV0AokqE

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
there is no native staking on the filecoin blockchain. they will take your FIL, invest them into their mining operations and pay you the returns to some extend

**Comment by Jackyap84** [comment]
is it safe to let third party to use my FIL and mine ?
